Hyper/Window.cs

```diff
@@ -132,15 +112,46 @@ protected override void OnUpdateFrame(FrameEventArgs e)
             UpdateProjectiles(time);
         }
 
-        protected override void OnKeyDown(KeyboardKeyEventArgs e)
+        protected override void OnMouseDown(MouseButtonEventArgs e)
         {
-            base.OnKeyDown(e);
+            base.OnMouseDown(e);
 
-            if (e.Key == Keys.J)
+            if (e.Button == MouseButton.Middle)
             {
                 var projectile = new Projectile(CubeMesh.Vertices, _camera.ReferencePointPosition + 1 / _scale * Vector3.UnitY, _camera.Front, 20f, 5f);
                 _projectiles.Add(projectile);
             }
+
+            // These 2 do not work on chunk borders.
+            if (e.Button == MouseButton.Left)
+            {
+                var position = _camera.ReferencePointPosition;
+
+                //for (int i = 0; i < 10; i++)
```

Comment:
Either remove this or uncomment. I think uncomment for mining but remove dor building

Reply:
I removed for both to keep it consistent. We can change that later when working on players experience